partials/forms/select.html

```diff
@@ -1,5 +1,9 @@
 <div id="{{id}}-group" class="form-group{{#compound}} form-group-compound{{/compound}}{{#error}} validation-error{{/error}}">
-    <label for="{{id}}" class="{{labelClassName}}">{{label}}</label>
+    <label for="{{id}}" class="{{labelClassName}}">
+        {{#error}}<span id="{{key}}-error" class="error-message">{{error.message}}</span>{{/error}}
```

Comment:
You can remove the `id` attribute. The error message is part of the label, which is already associated with the select field. 

Please also add `aria-hidden="true"` otherwise screen reader uses will get the error message twice.